crates/context/config/src/client/near.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::exhaustive_structs, reason = "TODO: Allowed until reviewed")]
```

Comment:
you have a todo here

Reply:
Yup. That's for the code owner to pick up on and decide as to which should be exhaustive and which not.